frontend/packages/console-shared/src/constants/common.ts

```diff
@@ -102,3 +102,7 @@ export enum REQUESTER_FILTER {
 
 export const CLUSTER_VERSION_DEFAULT_UPSTREAM_SERVER_URL_PLACEHOLDER =
   'https://api.openshift.com/api/upgrades_info/v1/graph';
+
+export const GETTING_STARTED_USER_SETTINGS_KEY_ADD_PAGE = 'devconsole.addPage.gettingStarted';
+export const GETTING_STARTED_USER_SETTINGS_KEY_CLUSTER_DASHBOARD =
+  'console.clusterDashboard.gettingStarted';
```

Comment:
One more thing. Neighter the const name nor the user settings (ConfigMap) key helps the user understand what's saved here. Please add something like _IS_EXPANDED / _COLLAPSES or so.

Reply:
Updated the PR for this change